docs/addons/addon-migration-guide.mdx

```diff
@@ -17,79 +17,120 @@ Begin by updating your Storybook dependencies. Use the `next` tag for pre-releas
 
 ```jsonc title="package.json"
 {
-  "dependencies": {
-    "@storybook/client-logger": "next" // or "latest", or "^8.0.0"
+  "devDependencies": {
+    "storybook": "next" // or "latest", or "^9.0.0"
   }
 }
 ```
 
-## Key changes for addons
+### Dependency Management
 
-Here are the essential changes in version 8.0 that impact addon development. Please check the [complete migration note](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-7x-to-800) for an exhaustive list of changes in 8.0.
+With Storybook 9.0, most Storybook packages have been consolidated into the main `storybook` package. This means you no longer need to reference individual Storybook packages as dependencies. Instead, define `storybook` as a peer dependency in your addon's `package.json`:
 
-### Node.js 16 support dropped
+```jsonc title="package.json"
+{
+  "name": "your-storybook-addon",
+  "peerDependencies": {
+    "storybook": "^9.0.0"
+  },
+  "devDependencies": {
+    "storybook": "^9.0.0" // For local development
```

Comment:
Should this be `9.0.0-0.0` or something to include all prereleases?

Reply:
Good point!